src/ast2ram/ClauseTranslator.h

```diff
@@ -32,17 +32,28 @@ namespace souffle::ast2ram {
 
 class TranslatorContext;
 
+/** Translation modes */
+enum TranslationMode { DEFAULT, SubsumeRNC, SubsumeRNN, SubsumeDCD, SubsumeDCC };
```

Comment:
Q: Can we write
```
enum TranslationMode {
    DEFAULT,
    SubsumeRNC,  // explanation of RNC
    SubsumeRNN,  // ...
    SubsumeDCD,  // ...
    SubsumeDCC,  // ...
};
```
To explain what each of these mean? Am assuming `R` = reject, `D` = delete, `N` = new knowledge, but would like to clarify.

Reply:
Done.